### DISCUSSION TITLE
Error when opening RTL Lightning Wallet - Error: Connect Econnrefused ::1:10080

### DISCUSSION THREAD

**Discussion by RowdyMuncher** [Ride-The-Lightning/RTL#1344] [Troubleshooting Questions]
When logging in to RTL Lightning Wallet for the first time from mynode I get the following error. Error: Connect Econnrefused ::1:10080

Any help would be appreciated.
Cheers

**Comment by saubyk** [comment]
Hi @RowdyMuncher is your lightning node working? in other words is LND online and active?

**Comment by RowdyMuncher** [comment]
Hi @saubyk Under service status LDN is Running and active.

**Reply by RowdyMuncher** [reply]
I thought at first that this may have to do with a port that I have not opened as I am running a TP-Link Omada Network which as far as I can work out does not support NAT-PMP so I have had to open ports to get parts of the mynode working. Just a thought.

**Reply by saubyk** [reply]
I am not familiar with the network architecture you described. As long as the RTL server and LND node are running on the same device, your network configuration shouldn't matter.

Are you able to get to the login page? If yes, the part that you need to troubleshoot is RTL unable to talk to LND for some reason

**Reply by RowdyMuncher** [reply]
Yes I am able to get to the login page. After signing in I get the error. Attached is the screenshot. I understand that RTL is unable ti talk to LND what I am trying to ascertain if you or anyone knows is what port numbers need to be forwarded for RTL to talk to LND as I am pretty sure that that is the issue. Thanks for your time helping its very much appreciated.
![Capto_Capture 2024-01-17_11-50-27_am](https://github.com/Ride-The-Lightning/RTL/assets/44824397/6f82cf55-e544-4481-a2f2-86d5598c7d2d)
Cheers

**Reply by saubyk** [reply]
I am curious if LND is running on a device separate from RTL?
In any case RTL needs to be able to access LND's rest port, which should be configured with the value of `restlisten` parameter in lnd.conf

**Comment by richardsj** [comment]
Appears to be related to #1120 - i.e. 'localhost' needs to be replaced by changing lnServerUrl to 127.0.0.1 (from localhost) in RTL config on some implementations, e.g. that deployed on MyNoteBTC

**Comment by cstahmer1** [comment]
In settings > firewall there should be an option to "ALLOW ALL NETWORK TRAFFIC".  I was trying all sorts of things in the lnd.config file to get past this error.  Before resetting config in the lnd.config file I turned this setting on.  Tried again no luck.  I then reset config in the lnd.config and RTL was able to connect.
